components/Input/index.d.ts

```diff
@@ -41,4 +41,15 @@ export interface InputProps extends InputHTMLAttributes<HTMLInputElement> {
   };
 }
 
-export default class Input extends Component<InputProps> {}
+interface CompoundedInput extends React.FunctionComponent<InputProps> {
+  CEP: React.FunctionComponent<InputProps>;
+  CNPJ: React.FunctionComponent<InputProps>;
+  CPF: React.FunctionComponent<InputProps>;
+  Date: React.FunctionComponent<InputProps>;
+  Password: React.FunctionComponent<InputProps>;
```

Comment:
How about saying that this subcomponent has all the properties of the Input component except the "type" property?